Functions/CatalogItems/Write-RsCatalogItem.ps1

```diff
@@ -123,7 +124,7 @@ function Write-RsCatalogItem
                 $enabled = $content.DataSourceDefinition.Enabled
                 $credentialRetrieval = 'None'
 
-                $newDataSourceCmd = "New-RsDataSource -Destination $Destination -Name $itemName -Extension $extension -CredentialRetrieval $credentialRetrieval"
+                $newDataSourceCmd = "New-RsDataSource -RsFolder $RsFolder -Name $itemName -Extension $extension -CredentialRetrieval $credentialRetrieval"
```

Comment:
>New-RsDataSource [](start = 37, length = 16)

I don't see this function modified and its current version expects Destination, not RsFolder as a parameter. Could you please make changes to this function too?